cluster/examples/vmi-sriov.yaml

```diff
@@ -19,8 +19,10 @@ spec:
         volumeName: cloudinitvolume
       interfaces:
       - bridge: {}
+        dhcpOptions: {}
         name: default
-      - name: sriov-net
+      - dhcpOptions: {}
```

Comment:
Ah that looks suspicios. The struct is not formed correctly. See the comment below.